src/elm/Claim.elm

```diff
@@ -389,6 +387,242 @@ viewClaimCard { shared, accountName } profileSummary claim =
+                        |> Utils.posixDateTime
+                        |> Strftime.format "%d %b %Y" Time.utc
+
+                ( claimStatusPhrase, claimStatus, textColor ) =
+                    case claim.status of
+                        Approved ->
+                            ( t "claim.title_approved.1", t "claim.approved", "text-2xl font-bold lowercase text-green" )
+
+                        Rejected ->
+                            ( t "claim.title_rejected.1", t "claim.disapproved", "text-2xl font-bold lowercase text-red" )
+
+                        Pending ->
+                            ( t "claim.title_under_review.1", t "claim.pending", "text-2xl font-bold lowercase text-gray-600" )
+
+                claimDate =
+                    let
+                        claimedOn =
+                            t "claim.claimed_on"
+
+                        dateTime =
+                            date claim.createdAt
+                    in
+                    claimedOn ++ " " ++ dateTime
+            in
+            div [ class "block" ]
+                [ p [ class "text-xs uppercase" ] [ text claimDate ]
+                , label [ class "text-2xl font-bold" ]
+                    [ text claimStatusPhrase
+                    ]
+                , div [ class textColor ] [ text claimStatus ]
+                ]
+
+        rewardInfo =
+            let
+                rewardTxtStyle =
+                    "my-2 font-bold text-lg"
+            in
+            div
+                [ class "text-center flex justify-center bg-gray-100 rounded-md p-4 space-x-16" ]
+                [ div
+                    []
+                    [ p [ class rewardTxtStyle ] [ text (String.fromFloat claim.action.reward) ]
+                    , p [ class greenTextTitleStyle ] [ text (t "community.actions.reward") ]
+                    ]
+                , div []
+                    [ p [ class rewardTxtStyle ] [ text (String.fromFloat claim.action.verifierReward) ]
+                    , p [ class greenTextTitleStyle ] [ text (t "claim.analyst_reward") ]
+                    ]
+                ]
+
+        approvedBy =
+            div
+                [ class claimVerifiersList ]
+                [ p [ class greenTextTitleStyle ] [ text (t "claim.approved_by") ]
+                , div [ class "overflow-x-auto" ]
+                    [ if List.isEmpty claim.checks then
+                        div [ class "flex mb-10" ]
+                            [ div [ class "pt-2" ] [ Profile.viewEmpty loggedIn.shared ]
+                            ]
+
+                      else
+                        div [ class "flex -mx-2 pt-2" ]
+                            (List.map
+                                (\c ->
+                                    if c.isApproved then
+                                        div [ class "px-2" ]
+                                            [ Profile.view loggedIn.shared loggedIn.accountName c.validator
```

Comment:
After #537, Profile.view and Profile.viewLarge have been replaced by the functions in Profile.Summary - you need to keep track of a Profile.Summary.Model and have it update on your update function (this is that nested TEA thing you were doing on Neutro!)

This one is trickier because it's actually multiple of them. [Here](https://github.com/cambiatus/frontend/blob/717c820a54df355498f257cb5efa33ffb363f8f3/src/elm/Page/Dashboard/Analysis.elm#L345-L347) is an example with multiple of them being handled